extensions/resteasy-common/deployment/src/main/java/io/quarkus/resteasy/common/deployment/ResteasyDotNames.java

```diff
@@ -63,19 +63,23 @@
     private static class IgnoreForReflectionPredicate implements Predicate<DotName> {
 
         @Override
-        public boolean test(DotName name) {
-            return ResteasyDotNames.TYPES_IGNORED_FOR_REFLECTION.contains(name)
-                    || ReflectiveHierarchyBuildItem.DefaultIgnorePredicate.INSTANCE.test(name);
+        public boolean test(DotName dotName) {
+            if (ResteasyDotNames.TYPES_IGNORED_FOR_REFLECTION.contains(dotName)
+                    || ReflectiveHierarchyBuildItem.DefaultIgnorePredicate.INSTANCE.test(dotName)) {
+                return true;
+            }
+            String name = dotName.toString();
+            for (String packageName : PACKAGES_IGNORED_FOR_REFLECTION) {
+                if (name.startsWith(packageName)) {
+                    return true;
+                }
+            }
+            return false;
         }
     }
 
     // Types ignored for reflection used by the RESTEasy and SmallRye REST client extensions.
     private static final Set<DotName> TYPES_IGNORED_FOR_REFLECTION = new HashSet<>(Arrays.asList(
-            // javax.json
-            DotName.createSimple("javax.json.JsonObject"),
-            DotName.createSimple("javax.json.JsonArray"),
-            DotName.createSimple("javax.json.JsonValue"),
```

Comment:
Wouldn't these also need to go away?

Reply:
Nevermind, my screen practically doesn't show the color GH uses to mark removal...